src/java/picard/sam/MergeSamFiles.java

```diff
@@ -52,7 +52,17 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Merges multiple SAM/BAM files into one file.",
+        usage = "Merges multiple SAM/BAM files into one file." +
```

Comment:
Perhaps also useful to find out what happens to the header information (are there rules for combining header information).